test/tests-crud-hooks.js

```diff
@@ -263,7 +263,8 @@ module("crud-hooks", {
 const expect = async (function* (expected, modifyer) {
     yield reset();
     yield modifyer();
-    equal(JSON.stringify(opLog, null, 2), JSON.stringify(expected, null, 2), "Expected oplog: " + JSON.stringify(expected));
+    equal(JSON.stringify(opLog, null, 2), JSON.stringify(
+        expected.map((x) => Object.assign({}, x, {updatedObj: undefined})), null, 2), "Expected oplog: " + JSON.stringify(expected));
```

Comment:
Could be change to {...x, updatedObj: undefined} for transpiled tests to work on IE11.

Reply:
Ha! Originally I used the spread operator but changed it to Object.assign not realizing it was transpiled. Pushing change now...